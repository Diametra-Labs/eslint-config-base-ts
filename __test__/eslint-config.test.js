const eslintConfig = require('../index');

describe('The configuration file should not change', () => {
  test('It should response with custom configuration', () => {
    expect(eslintConfig).toMatchSnapshot();
  });
});
