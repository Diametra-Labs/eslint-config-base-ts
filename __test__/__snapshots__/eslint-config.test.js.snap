// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The configuration file should not change It should response with custom configuration 1`] = `
Object {
  "env": Object {
    "browser": true,
    "es6": true,
    "node": true,
  },
  "extends": Array [
    "eslint:recommended",
    "plugin:@typescript-eslint/eslint-recommended",
    "plugin:@typescript-eslint/recommended",
    "plugin:react/recommended",
    "prettier/@typescript-eslint",
    "plugin:prettier/recommended",
  ],
  "parser": "@typescript-eslint/parser",
  "parserOptions": Object {
    "ecmaFeatures": Object {
      "jsx": true,
    },
    "ecmaVersion": 11,
    "sourceType": "module",
  },
  "plugins": Array [
    "@typescript-eslint",
    "react-hooks",
  ],
  "root": true,
  "rules": Object {
    "@typescript-eslint/explicit-function-return-type": "off",
    "prettier/prettier": Array [
      "error",
      Object {
        "singleQuote": true,
      },
    ],
    "react-hooks/exhaustive-deps": "warn",
    "react-hooks/rules-of-hooks": "error",
    "react/react-in-jsx-scope": "off",
  },
}
`;
